### PR TITLE
[Bulk] Validation-OData-ms-Author-invalid1

### PR DESCRIPTION
**Pull request by Roudabeh1348**
This PR fixes 30 issues of invalid ms.author and missing descriptions in Odata-docs.

### CODE REVIEW

**Comment by Roudabeh1348** [comment]
@xuzhg The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. 
This will eliminate potential accessibility, security, and usability issues. 
This PR includes only build validation fixes and does not change other content. 

Please review within five business days and merge or comment in the PR with any changes you’d like to see. 
Thank you!

**Comment by Roudabeh1348** [comment]
@xuzhg @odero @mikepizzo This is a friendly reminder that this PR has been open and submitted for review since last week, I appreciate it if you could review the PR to either sign-off and merge or advise me of any further issues you might find or any other changes you'd wish me to make to this PR.
Thank you!

**Comment by odero** [comment]
@habbes, @KenitoInc The added descriptions seem off. Please review when you get a chance

File: Odata-docs/odatalib/v7/simplified-options.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Centralized control for OData Simplified Options"
-description: ""
+description: "From ODataV7.0, we add a centralized control class for them, which is ODataSimplifiedOptions."
```
Comment (KenitoInc) [inline Odata-docs/odatalib/v7/simplified-options.md:3]:
```suggestion
description: "This section describes how to use ODataSimplifiedOptions."
```

**Review by KenitoInc — CHANGES_REQUESTED** [review]

File: Odata-docs/odatalib/v7/capabilities-vocabulary.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Capabilities vocabulary support in odatalib v7"
-description: ""
+description: "From ODataLib 6.13.0, it supports the capabilities vocabulary. The capabilities vocabulary is enabled as a reference model in the Edm Model."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/capabilities-vocabulary.md:3]: revised and replaced with the suggested description.

File: Odata-docs/odatalib/v7/customizable-type-facets-promotion.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Customizable type facets promotion in URI parsing in odatalib"
-description: ""
+description: "Class ODataUri has two properties Filter and OrderBy that are tree data structures representing part of the URI parsing result. When an operation is applied to types with different facets, they will first be converted to a common type with identical facets."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/customizable-type-facets-promotion.md:3]: revised and replaced with the suggested description.

File: Odata-docs/odatalib/v7/di-support.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
 title: " Dependency Injection Support"
-description: ""
+description: "In ODataLib v7.0, Dependency Injection support was introduced to dramatically increase the extensibility of the library where users can plug in their custom implementations and policies in an elegant way."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/di-support.md:3]: revised and replaced with the suggested description.

File: Odata-docs/odatalib/v7/edm/define-singleton.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
 title: " Define singletons"
-description: "Define singleton using EdmLib APIs"
+description: "Define singleton using EdmLib APIs. Defining a singleton in the entity container shares the same simple way as defining an entity set."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/edm/define-singleton.md:3]: revised and replaced with the suggested description.

File: Odata-docs/odatalib/v7/expanded-navigation-delta.md [hunk 6]
```diff
@@ -1,7 +1,8 @@
 ---
 title: "Expanded Navigation Property in ODL Delta Response"
+description: "In ODataLib 6.15.0, we introduced the support for reading and writing expanded navigation properties (either collection or single) in delta responses."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/expanded-navigation-delta.md:3]: revised and replaced with the suggested description.

File: Odata-docs/odatalib/v7/override-primitive-serialization-deserialization.md [hunk 7]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Override payload serialization and deserialization in odatalib"
-description: ""
+description: "ODataLib 6.12.0 supports to customize the payload value converter to override the primitive serialization and deserialization of payload."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/override-primitive-serialization-deserialization.md:3]: revised and replaced with the suggested description.

File: Odata-docs/odatalib/v7/optional-parameters.md [hunk 8]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Optional Parameters"
-description: ""
+title: "Optional parameters in ODataLib 7.3"
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/optional-parameters.md:2]: revised and replaced with the suggested description.

File: Odata-docs/odatalib/v7/parsing-url-template.md [hunk 9]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Parsing uri path template"
-description: ""
+description: "OData uri parser can parse uri path template. A path template is any identifier string enclosed with curly braces."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/parsing-url-template.md:3]: revised and replaced with the suggested description.

File: Odata-docs/odatalib/v7/read-payload.md [hunk 10]
```diff
@@ -1,8 +1,8 @@
 ---
 title: " Read OData payload"
-description: "Read OData payload using OData Core APIs"
+description: "Read OData payload using OData Core APIs. The reader API is similar to the writer API, and you can expect symmetry here."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/read-payload.md:3]: revised and replaced with the suggested descriptions.

File: Odata-docs/odatalib/v7/simplified-options.md (hunk 1 above)
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/simplified-options.md:3]: revised and replaced with the suggested description.

**Comment by Roudabeh1348** [comment]
@habbes @KenitoInc the suggested revisions for descriptions were made to the mentioned articles in this PR.

**Comment by KenitoInc** [comment]
@Roudabeh1348 It seems the files below haven't been updated with the changes requested
json-batch.md
navigation-partner.md
nextpage-link-and-count-for-collection.md

File: Odata-docs/odatalib/v7/edm/define-singleton.md [hunk 11]
```diff
@@ -1,8 +1,8 @@
 ---
 title: " Define singletons"
-description: "Define singleton using EdmLib APIs"
+description: "This section describes how to define singleton using EdmLib APIs. Defining a singleton in the entity container shares the same simple way as defining an entity set."
```
Comment (odero) [inline Odata-docs/odatalib/v7/edm/define-singleton.md:3]:
```suggestion
description: "This section describes how to define singletons using EdmLib APIs. Defining a singleton in the entity container shares the same simple way as defining an entity set."
```

**Review by odero — CHANGES_REQUESTED** [review]

File: Odata-docs/odatalib/v7/edm/define-singleton.md (hunk 11 above)
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/edm/define-singleton.md:3]: fixed.

**Comment by KenitoInc** [comment]
> @Roudabeh1348 It seems the files below haven't been updated with the changes requested
json-batch.md
navigation-partner.md
nextpage-link-and-count-for-collection.md

Did you resolve this? @Roudabeh1348

File: Odata-docs/odatalib/v7/json-batch.md [hunk 12]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Json batch format in ODataLib"
-description: ""
+description: "OData supports grouping multiple requests together, sending a single HTTP request to OData service, and grouping corresponding responses together before sending back as a single HTTP response."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/json-batch.md:3]: fixed.

File: Odata-docs/odatalib/v7/navigation-partner.md [hunk 13]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Navigation property partner"
-description: ""
+description: "The library supports setting and retrieving partner information of navigation properties."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/navigation-partner.md:3]: fixed.

File: Odata-docs/odatalib/v7/nextpage-link-and-count-for-collection.md [hunk 14]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Write NextPageLink/Count for collection - odatalib"
-description: ""
+description: "From ODataLib 6.13.0, it supports to write the NextPageLink/Count instance annotation in top-level collection payload."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/nextpage-link-and-count-for-collection.md:3]: fixed.

File: Odata-docs/odatalib/v7/fluent-writer-api.md [hunk 15]
```diff
@@ -1,8 +1,8 @@
 ---
 title: " Fluent functional-style writer API"
-description: ""
+description: "In this version, a new set of fluent functional-style API has been introduced as an improvement over the previous API which is rather primitive."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/fluent-writer-api.md:3]: fixed.

File: Odata-docs/odatalib/v7/json-batch.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/json-batch.md:3]: fixed.

File: Odata-docs/odatalib/v7/navigation-partner.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/navigation-partner.md:3]: fixed.

File: Odata-docs/odatalib/v7/nextpage-link-and-count-for-collection.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/nextpage-link-and-count-for-collection.md:3]: fixed.

**Comment by Roudabeh1348** [comment]
> > @Roudabeh1348 It seems the files below haven't been updated with the changes requested
> > json-batch.md
> > navigation-partner.md
> > nextpage-link-and-count-for-collection.md
> 
> Did you resolve this? @Roudabeh1348

Yes, I fixed the descriptions as per your suggestions. Please let me know if there are any other issues with the PR.

**Comment by Roudabeh1348** [comment]
The requested changes were made to the descriptions. Let me know if there is any other issue that needs to be resolved.

**Review by KenitoInc — APPROVED** [review]
LGTM

**Review by xuzhg — APPROVED** [review]

**Review by xuzhg — APPROVED** [review]

**Pull request merged**